### DISCUSSION TITLE
北捷5G運輸計畫🚄

### DISCUSSION THREAD

**Discussion by bonny123865** [1101-GS3073-C/Python-Community-GS3073-C#110] [General]
**新聞大綱:**
臺北捷運公司與大同世界科技（大世科）、伸波通訊等業者合作，推動5G運輸智慧監控平臺計畫，結合5G專網與AI影像辨識技術，辨識車廂內旅客動態，包括是否有人員異常的行為，除了用於維護車廂內安全，該計畫也可能和大同大學合作，以AI軌道巡檢技術確保軌道安全。
先選定臺北捷運松山線的中山站、松江南京、南京復興三站，由伸波通訊與和碩合作，以微軟Affirmed Networks核心網路與和碩的5G ORAN，建置小型5G獨立專網，預計明年9月完成。
在全長2.4公里地下路段建置5G專網，為不影響捷運日常營運，只在停班時間下進行建置，因應高畫質車廂監控影像的傳輸需要，該專網上傳速度經調整，可進一步提升至150Mbps，有助於監控影像的即時傳輸。
未來結合5G的高速傳輸，可將車廂內影像傳到南京復興站的邊緣運算機房，該機房透過Nvidia DGX進行AI影像辨識，再將異常事件的辨識結果送到行控中心告警。
臺北捷運表示，5G運輸智慧監控平臺計畫有三項特色
1.	服務採模組化，以利於未來快速複製到其他的軌道產業，甚至是輸出至國外
2.	國產整合方案，結合國內業者發展的自主通訊技術
3.	中高運量路段測試，未來可擴大至臺北捷運場域，服務每日上百萬人次。

**新聞心得:**
利用監控車內即時影像並立即回報、AI軌道巡檢技術，除了讓車廂內的乘客能更安心地知道自己目前搭乘的乘車狀況，也讓駕駛艙、捷運中心人員能夠更準確地了解詳情，對於維修設備也更添加一份保障。雖然為了因應捷運日常運行而採用停班時期實施工程，相對會比較拖延完成時間，但也很希望這項計畫能夠如期完成，讓乘客有一個更安全、更高科技的乘車體驗。
**新聞來源:**
https://www.ithome.com.tw/news/147642

**Comment by Timmy110403540** [comment]
讓我想到幾天前的新聞:日本九州新幹線放火事件以及之前北捷的慘案
望能利用此科技快速掌握車廂內的動態，給予乘客一定的安全感!

**Comment by pocper1** [comment]
AI 影像辨識的技術應該還有雪山隧道慢速車示警系統，我想也是透過AI影像辨識車牌~
新聞網址:[雪隧「路隊長」退散！慢速車示警系統增3處 10/8啟用](https://news.ltn.com.tw/news/life/breakingnews/3682428)

**Comment by ghost** [comment]
技術很酷 期待真正實測安裝的結果~

**Comment by LanningYang** [comment]
AI 科技執法 能減少警消人員的傷亡
